scripts/cmake/Modules/PackagingFedora.cmake

```diff
@@ -19,15 +19,15 @@ set (CPACK_RPM_PACKAGE_AUTOREQPROV 1)
 
 set (CPACK_RPM_CHANGELOG_FILE "${CMAKE_SOURCE_DIR}/scripts/packaging/fedora/changelog")
 
-execute_process (COMMAND bash "${CMAKE_SOURCE_DIR}/scripts/packaging/fedora/map_licenses.sh" "${CMAKE_SOURCE_DIR}/doc/THIRD-PARTY-LICENSES"
+execute_process (COMMAND bash "${CMAKE_SOURCE_DIR}/scripts/packaging/fedora/map_licenses.sh" "${CMAKE_SOURCE_DIR}/.reuse/dep5"
 		 OUTPUT_VARIABLE THIR_PARTY_LICENSES_STR)
-set (CPACK_RPM_PACKAGE_LICENSE "${THIR_PARTY_LICENSES_STR} \n # For a breakdown of the licensing, see THIRD-PARTY-LICENSES")
+set (CPACK_RPM_PACKAGE_LICENSE "${THIR_PARTY_LICENSES_STR} \n # For a breakdown of the licensing, see .reuse/dep5")
 # install license files
 configure_file ("${CMAKE_SOURCE_DIR}/LICENSE.md" "${CMAKE_BINARY_DIR}/doc/LICENSE" COPYONLY)
-configure_file ("${CMAKE_SOURCE_DIR}/doc/THIRD-PARTY-LICENSES" "${CMAKE_BINARY_DIR}/doc/THIRD-PARTY-LICENSES" COPYONLY)
+configure_file ("${CMAKE_SOURCE_DIR}/.reuse/dep5" "${CMAKE_BINARY_DIR}/.reuse/dep5" COPYONLY)
```

Comment:
I am not sure if we should really install this file under the name `dep5`. A few other rpm packages like checkbashism and devscripts include a dep5 license file as well, but they are naming it `copyright`. Maybe we should install it under the name `copyright` as well, similar to our generated Debian packages.